src/Kovan-DssSpell.sol

```diff
@@ -28,45 +38,255 @@ contract SpellAction {
+
+        // Update UNI-A spot value in Vat
+        SpotAbstract(MCD_SPOT).poke(ILK_UNI_A);
+
+        // Add new ilk to the IlkRegistry
+        IlkRegistryAbstract(ILK_REGISTRY).add(MCD_JOIN_UNI_A);
+
+        // Set gulp amount in faucet on kovan (only use WAD for decimals = 18)
+        FaucetAbstract(FAUCET).setAmt(UNI, 2500 * WAD);
+
+        // Update the changelog
+        CHANGELOG.setAddress("UNI", UNI);
+        CHANGELOG.setAddress("MCD_JOIN_UNI_A", MCD_JOIN_UNI_A);
+        CHANGELOG.setAddress("MCD_FLIP_UNI_A", MCD_FLIP_UNI_A);
+        CHANGELOG.setAddress("PIP_UNI", PIP_UNI);
+
+        //
+        // Add renBTC
+        //
 
-        // Set new Chief in the Pause
-        DSPauseAbstract(MCD_PAUSE).setAuthority(MCD_ADM);
+        // Add RENBTC-A ilk
+        require(GemJoinAbstract(MCD_JOIN_RENBTC_A).vat() == MCD_VAT, "join-vat-not-match");
+        require(GemJoinAbstract(MCD_JOIN_RENBTC_A).ilk() == RENBTC_ILK, "join-ilk-not-match");
+        require(GemJoinAbstract(MCD_JOIN_RENBTC_A).gem() == RENBTC, "join-gem-not-match");
+        require(GemJoinAbstract(MCD_JOIN_RENBTC_A).dec() == DSTokenAbstract(RENBTC).decimals(), "join-dec-not-match");
+        require(FlipAbstract(MCD_FLIP_RENBTC_A).vat() == MCD_VAT, "flip-vat-not-match");
+        require(FlipAbstract(MCD_FLIP_RENBTC_A).cat() == MCD_CAT, "flip-cat-not-match");
+        require(FlipAbstract(MCD_FLIP_RENBTC_A).ilk() == RENBTC_ILK, "flip-ilk-not-match");
 
-        // Set new Chief in the FlipperMom
-        FlipperMomAbstract(FLIPPER_MOM).setAuthority(MCD_ADM);
+        SpotAbstract(MCD_SPOT).file(RENBTC_ILK, "pip", PIP_RENBTC);
 
-        // Set new Chief in the OsmMom
-        OsmMomAbstract(OSM_MOM).setAuthority(MCD_ADM);
+        // Set the RENBTC-A Flipper in the Cat
+        CatAbstract(MCD_CAT).file(RENBTC_ILK, "flip", MCD_FLIP_RENBTC_A);
+
+        // Init RENBTC-A ilk in Vat & Jug
+        VatAbstract(MCD_VAT).init(RENBTC_ILK);
+        JugAbstract(MCD_JUG).init(RENBTC_ILK);
+
+        // Allow RENBTC-A Join to modify Vat registry
+        VatAbstract(MCD_VAT).rely(MCD_JOIN_RENBTC_A);
+        // Allow the RENBTC-A Flipper to reduce the Cat litterbox on deal()
+        CatAbstract(MCD_CAT).rely(MCD_FLIP_RENBTC_A);
+        // Allow Cat to kick auctions in RENBTC-A Flipper
+        FlipAbstract(MCD_FLIP_RENBTC_A).rely(MCD_CAT);
+        // Allow End to yank auctions in RENBTC-A Flipper
+        FlipAbstract(MCD_FLIP_RENBTC_A).rely(MCD_END);
+        // Allow FlipperMom to access to the RENBTC-A Flipper
+        FlipAbstract(MCD_FLIP_RENBTC_A).rely(FLIPPER_MOM);
+        // Disallow Cat to kick auctions in RENBTC-A Flipper
+        // !!!!!!!! Only for certain collaterals that do not trigger liquidations like USDC-A)
+        // FlipperMomAbstract(FLIPPER_MOM).deny(MCD_FLIP_RENBTC_A);
+
+        // Allow OsmMom to access to the RENBTC Osm
+        // !!!!!!!! Only if PIP_RENBTC = Osm and hasn't been already relied due a previous deployed ilk
```

Comment:
To confirm, we're using the existing WBTC OSM here, right?

Reply:
yes.  @NiklasKunkel might want to also confirm this was what the oracle team wanted.

Reply:
confirmed, we'll use the WBTC OSM